website/docs/getting-started/index.mdx

```diff
@@ -403,27 +393,21 @@ export class PostsComponent {
 
 Now, with simple configuration and an npm/yarn script, a front-end developers benefits from:
 
+- **up-to-date typings**
 
-  - **up-to-date typings**
-
-
-  - **autocompletion on all queries, mutations and, subscription variables and results**
-
-
-  - **less boilerplate** (thanks to full code generation such as React hooks generation)
-
+- **autocompletion on all queries, mutations and, subscription variables and results**
```

Comment:
Actually not true in Svelte.
Need https://github.com/ardatan/graphql-tools/pull/4349 and a PR (that I will do now on vscode)